ariadne/graphql.py

```diff
@@ -114,59 +114,73 @@ def graphql_sync(
     validation_rules: Optional[Sequence[RuleType]] = None,
     error_formatter: ErrorFormatter = format_error,
     middleware: Optional[MiddlewareManager] = None,
+    extensions: Optional[List[Type[Extension]]] = None,
     **kwargs,
 ) -> GraphQLResult:
-    try:
-        validate_data(data)
-        query, variables, operation_name = (
-            data["query"],
-            data.get("variables"),
-            data.get("operationName"),
-        )
+    extension_manager = ExtensionManager(extensions)
+    with extension_manager.request(context_value):
```

Comment:
In `graphql` there's an empty line between extension_manager and context: 

```suggestion

    with extension_manager.request(context_value):
```